DrinkManagerWeb/Views/Drinks/Index.cshtml

```diff
@@ -0,0 +1,51 @@
+﻿@model BLL.PaginatedList<BLL.Drink>
+
+@{
+    ViewBag.Title = "Drinks";
+    Layout = "_Layout";
+}
+
+<p>
+    <a asp-action="Create">Create New</a>
+</p>
+<table class="table">
+    <thead>
+        <tr>
+            <th>
+                <a asp-action="Index" asp-route-sortOrder="@ViewData["NameSortParm"]" asp-route-currentFilter="@ViewData["CurrentFilter"]">Name</a>
+            </th>
+            <th></th>
+        </tr>
+    </thead>
+    <tbody>
+        @foreach (var drink in Model)
+        {
+                <tr>
+                    <td>
+                        <a asp-action="Details" asp-route-id="@drink.Id">@Html.DisplayFor(modelItem => drink.Name)</a>
+                    </td>
+                    <td>
+                        <img src="@drink.ImageUrl" width="100" height="100" asp-append-version="false" />
```

Comment:
width height attributes should not be used, we should use css classes instead

Reply:
Actually width/height attributes on the img element are totally acceptable, especially if it's used only in one place.

Reply:
I'v removed this code - visuals can be done later.